.github/workflows/ic-ui-kit-branches.yml

```diff
@@ -235,7 +235,7 @@ jobs:
         uses: JamesIves/github-pages-deploy-action@65b5dfd4f5bcd3a7403bbc2959c144256167464e #v4.5.0
         with:
           folder: ./packages/react/cypress-image-diff-html-report
-          target-folder: ./branches/${{ steps.extract_branch.outputs.branch }}/cypress-image-diff-html-report
+          target-folder: ./branches/${{ steps.extract_branch.outputs.branch }}/cypress-image-diff-html-report/cypress-image-diff-html-report.html
```

Comment:
This field is called 'target-folder' - does the value need to be changed to the file name?